tests/rest/admin/test_user.py

```diff
@@ -1222,6 +1224,17 @@ def test_deactivate_user_erase_true(self) -> None:
 
         self._is_erased("@user:test", True)
 
+        channel = self.make_request(
+            "GET",
+            self.url_users + "?deactivated=true",
+            access_token=self.admin_user_tok,
+        )
+
+        self.assertEqual(
+            1,
+            len(list(filter(lambda user: user["erased"], channel.json_body["users"]))),
+        )
```

Comment:
This is a little confusing. It might be more idomatic to write

```python
users = { user["name"]: user for user in channel.json_body["users"] }
```

and then `self.assertIs(users["@user:test"]["erased"], True)`.